packages/next/src/server/dev/hot-reloader-turbopack.ts

```diff
@@ -143,6 +143,14 @@ export async function createHotReloaderTurbopack(
       middlewareMatchers: undefined,
     }),
   })
+  // Intentionally not awaited as it starts a server
+  if (process.env.NEXT_TURBOPACK_TRACING) {
+    bindings.turbo.startTurbopackTraceServer(join(distDir, 'trace.log'))
```

Comment:
I think there should be a way to opt-out of that since it drains a lot of memory and performance.

Maybe `NEXT_TURBOPACK_TRACE_SERVER=0`?

Reply:
Or we make it opt-in with a flag instead